#! /bin/sh -e

# DP: Apply proposed fix for PR target/28102.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

2006-07-15  Alfred M. Szmidt  <ams@gnu.org>

       * config/i386/gnu.h (TARGET_C99_FUNCTIONS): Undefine macro.

Index: gcc/config/i386/gnu.h
===================================================================
--- gcc/config/i386/gnu.h       (revision 115472)
+++ gcc/config/i386/gnu.h       (working copy)
@@ -36,3 +36,5 @@

 /* FIXME: Is a Hurd-specific fallback mechanism necessary?  */
 #undef MD_UNWIND_SUPPORT
+
+#undef TARGET_C99_FUNCTIONS /* config.gcc includes i386/linux.h.  */


